app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionsImportFragment.java

```diff
@@ -177,7 +177,8 @@ public void onImportUrl(final String value) {
     }
 
     public void onImportFile() {
-        requestImportFileLauncher.launch(StoredFileHelper.getPicker(activity));
+        // leave */* mime type to support all services with different mime types and file extensions
+        requestImportFileLauncher.launch(StoredFileHelper.getPicker(activity, "*/*"));
```

Comment:
What is the difference between `"*/*"` and not passing the argument at all?

Reply:
Yes, there was no parameter at all before, mow I've added it